storage_logic | define goods, amount, weight

AGrzaNNa opened the pull request:
Completed the points for:

#define with points want the goods to be delivered to them and some points want boolean 1 | 0, and total weight

#created the size of the goods to be be delivered or collected (between 100kg and 200kg);

#random the amount of goods, their type and delivery or collect status

AGrzaNNa committed:
Completed points regarding:

# define with points want the goods to be delivered to them and some points want boolean 1 | 0, and total weight

#created the size of the goods to be be delivered or collected (between 100kg and 200kg);

#random the amount of goods, their type and delivery or collect status

AGrzaNNa committed:
Update README.md

AGrzaNNa committed:
Update README.md

gajdam committed:
fix: gitignore

gajdam reviewed (changes requested):
Move to data_generator

Commit:
Remove unnecessary WarehousesLogic class

Refactor generate_clients to change the logic for total_weight, delivery_type, and goods assignment.

Move distribute_weight method into DataGenerator class for better encapsulation.

Commit:
Revert "fix: gitignore"

This reverts commit 2866f091e410a4e7cca7df8f37b855a5b6af6020.

AGrzaNNa committed:
Update .gitignore

update gitignore with .vs

AGrzaNNa committed:
Delete .vs directory

Remove unnecessary files duo to lack of gitignore

gajdam commented on .gitignore:
```diff
@@ -171,7 +171,4 @@ cython_debug/
-.pypirc
-
-# vs code & vs
-.vs
```
add back .vs to gitigniore

AGrzaNNa replied:
gitignore updated

gajdam commented on .vs/PythonSettings.json:
to be removed

AGrzaNNa replied:
removed

AGrzaNNa replied:
removed

gajdam commented on .vs/delivery-optimizer/v16/.suo:
to be removed

AGrzaNNa replied:
removed

gajdam commented on .vs/slnx.sqlite:
to be removed

AGrzaNNa replied:
removed

bielansky commented on README.md:
Please delete this readme, this is going to be merged into main, we dont need information about your branch
